psi4/driver/p4util/text.py

```diff
@@ -256,10 +256,10 @@ def message_box(message: str = None, max_width: int = 80, min_width: int = 30):
 
     fmt = "! {:" + str(box_width) + "} !\n"
     for line in msg[:]:
-        error_str.extend([fmt.format(x) for x in wrap(line, box_width)])
+        error_str.extend([fmt.format(x) for x in wrap(line, box_width, subsequent_indent="    ")])
```

Comment:
@hokru this helps for longer errors. wasn't sure if it had been added elsewhere.

Reply:
Cool! I think you mentioned it to me in slack. At least I did not add it yet.